software/cnvkit/main.nf

```diff
@@ -0,0 +1,65 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+def options    = initOptions(params.options)
+
+def VERSION = '0.9.8'
+

```

Comment:
```suggestion
```

Reply:
change done.

Reply:
I don't think you actually pushed any of the change you did related to this comment